documentation/Changelog-for-next-generation.md

```diff
@@ -0,0 +1,13 @@
+The next-generation Azure JavaScript libraries introduce a few important changes:
+
+1. Authentication: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are no longer supported. Use package [@azure/identity](https://www.npmjs.com/package/@azure/identity) instead. Select a credential from Azure Identity examples based on the authentication method of your choice. You may find out the new authentication examples [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#authentication)  
```

Comment:
All numbers (for the points) are marked as 1. Please fix it

Reply:
That's just markdown format, if will show up as 1, 2, 3 if you are reading it in rich text mode. In that way, if we are inserting some items in the middle, we don't need to change all the numbers for the after items.